src/api/tests/common.py

```diff
@@ -409,3 +409,38 @@ def make_feature_flag(**kwargs):
 def make_supported_language(code=f"en-US-{datetime.now().timestamp()}", name="English (US)"):
     lang = SupportedLanguage.objects.create(code=code, name=name)
     return lang
+
+
+def create_supported_language(**kwargs):
+    code = kwargs.get("code") or f"en-US-{datetime.now().timestamp()}"
+    name = kwargs.get("name") or "English (US)"
+    lang = SupportedLanguage.objects.create(code=code, name=name)
+    return lang
+
+
+def make_campaign_account(**kwargs):
+    creator = kwargs.get("creator") or makeAdmin()
+    community = kwargs.get("community") or makeCommunity()
+    subdomain = kwargs.get("subdomain") or f"test.campaign.account-{datetime.now().timestamp()}"
+    account = CampaignAccount.objects.create(**{
+        "name": "Test Campaign Account",
+        "creator": creator,
+        "community": community,
+        "subdomain": subdomain,
+    })
+    return account
```

Comment:
You may want to just inline this return since `account` isn't being used anywhere else